dbms/tests/integration/test_multiversion_merge_tree_vertical_merge/configs/mergetree_settings.xml

```diff
@@ -0,0 +1,7 @@
+<yandex>
+     <merge_tree>
+                <enable_vertical_merge_algorithm>1</enable_vertical_merge_algorithm>
```

Comment:
We need a test for Horizontal case also

Reply:
Also, now we can use the following syntax:
```sql
CREATE TABLE test
(
    k Int8 DEFAULT 0, 
    i Int8, 
    s String
)
ENGINE = MergeTree
ORDER BY k
SETTINGS index_granularity = 3, enable_vertical_merge_algorithm = 1
```
So, I suppose, we can make an ordinary test instead of integration one.

Reply:
A replicated test would be also helpful.